crates/conductor_lib/src/keystore.rs

```diff
@@ -119,8 +119,11 @@ impl Keystore {
     /// This will query `passphrase_manager` immediately to set a passphrase for the keystore.
     pub fn new(
         passphrase_manager: Arc<PassphraseManager>,
-        hash_config: Option<PwHashConfig>,
+        mut hash_config: Option<PwHashConfig>,
     ) -> HcResult<Self> {
+        if hash_config.is_none() {
+            hash_config = test_hash_config()
+        }
```

Comment:
if we do let hash_config = hash_config.or_else(||test_hash_config) we could get rid of that mutable

Reply:
(also does this mean every invocation of this will result in using a test_hash if it is not provided?)

Reply:
Yes, but since we use `hash_config` twice, we would either need to call the .or_else twice or create another variable. I thought this would be the best option..

Reply:
If we change the name it should be to `fast_unsecure_hash` not default.